### ISSUE TITLE
get datetime when features ingested

### ISSUE THREAD

**Issue by BlackTrade** [feast-dev/feast#1579]
Hello guys!

Is it possible to get the `datetime` when the features were ingested?
something like that 

`client.get_online_features(feature_refs=['features_table:datetime'],
                                             entity_rows=[{'entity': entity}])`

**Comment by woop** [comment]
Hi @BlackTrade, we're not currently exposing timestamp information, but it should be pretty easy to expose as metadata in the response https://github.com/feast-dev/feast/blob/master/protos/feast/serving/ServingService.proto#L93

**Comment by woop** [comment]
Would unix epoch time be acceptable?

**Comment by BlackTrade** [comment]
Yes  unix epoch will be acceptable. I need ingestion time for debugging purposes. 
I need to validate/debug cases when I make a request to the Fest faster than putting data there

**Comment by Michie-hub** [comment]
Hi! I'd like to work on this issue. I’m going to investigate the current online serving flow and see how the feature ingestion timestamp can be exposed in the response. If this is still relevant, I’d be happy to submit a PR.
